packages/core/saved-objects/core-saved-objects-migration-server-internal/index.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { KibanaMigrator, buildActiveMappings, mergeTypes } from './src';
+export type { KibanaMigratorOptions } from './src';
+
+// these are only used for integration tests
+export {
+  bulkOverwriteTransformedDocuments,
+  closePit,
+  createIndex,
+  openPit,
+  calculateExcludeFilters,
+  checkForUnknownDocs,
+  waitForIndexStatus,
+  initAction,
+  cloneIndex,
```

Comment:
@mistic is that right?

Reply:
They can, but given these tests are importing the `kbnTestServer`, which even after the migration will require importing all of core, they can't live in the same package as the code

Reply:
The tests will have separate dependencies from the rest of the package, but you're right that any tests which rely on `kbnTestServer` will need to be run every time the distributable changes in any way, meaning it probably should be an independent package with as few tests as necessary as it will need to run on most changes. That being said, you shouldn't need to export a ton of stuff if you're running the Kibana distributable. Can we just move the tests which need to use `kbnTestServer` to their own package and keep the more internal-style tests which use a bunch of internal functions, in the implementation package?